### ISSUE TITLE
getPeers command shows IP twice

### ISSUE THREAD

**Issue by Vort** [yggdrasil-network/yggdrasil-go#468]
When I execute `yggdrasilctl.exe getPeers` command, result shows IP in two columns: as key and as value (ip):
```
                                        bytes_recvd     bytes_sent      endpoint                     ip                                      port  proto  uptime     
201:870:9573:4095:f4e6:dfde:cc68:c783   3940165         2131857         tcp://185.164.138.18:1001    201:870:9573:4095:f4e6:dfde:cc68:c783   7     tcp    19:38:33        
```
This was not happening with 0.3.5.

Also (as I can see), list is unordered, which makes changes tracking harder.

**Comment by neilalexander** [comment]
That looks like a possible bug in `yggdrasilctl`. The random display order happens because Go maps are unsorted by nature.

**Comment by neilalexander** [comment]
Fixed by #469.
